compiler/control/OMROptions.cpp

```diff
@@ -4797,6 +4798,7 @@ char *OMR::Options::_verboseOptionNames[TR_NumVerboseOptions] =
    "hookDetailsClassLoading",
    "hookDetailsClassUnloading",
    "sampleDensity",
+   "profiling"
```

Comment:
For verbose options, the option name has to match its position in `TR_VerboseFlags`, rather than being alphabetically sorted like the main option list. Here is matches the value for `TR_VerboseProfiling`.